stubs/docker-compose.yml

```diff
@@ -41,6 +42,19 @@ services:
             - 'sailmysql:/var/lib/mysql'
         networks:
             - sail
+#    pgsql:
+#        image: postgres:12
+#        ports:
+#            - '${FORWARD_DB_PORT:-5432}:5432'
+#        environment:
+#            PGPASSWORD: '${DB_PASSWORD}'
+#            POSTGRES_DATABASE: '${DB_DATABASE}'
+#            POSTGRES_USER: '${DB_USERNAME}'
+#            POSTGRES_PASSWORD: '${DB_PASSWORD:secret}'
```

Comment:
It complained about incorrect interpolation, this fixed it:
```suggestion
#            POSTGRES_PASSWORD: '${DB_PASSWORD:-secret}'
```

Reply:
In fact, `:` without `-` after it is [substring expansion](https://wiki.bash-hackers.org/syntax/pe#substring_expansion). Please apply the suggestion when you get to updating the PR, thanks :)